gdk4/Cargo.toml

```diff
@@ -25,7 +25,7 @@ v4_8 = ["ffi/v4_8", "v4_6"]
 features = ["dox"]
 
 [dependencies]
-bitflags = "1.0"
+bitflags = "1.3"
```

Comment:
Why?

Reply:
isn't using 1.0 != 1.3?

Reply:
cargo will automatically pull in 1.3 if 1.0 is set here. But unless we use features that were added in 1.0 (we probably don't) then keeping it 1.0 is more correct as it allows using an old version if someone really insists.

Reply:
That is, 1.3 is considered backwards compatible to 1.0.